src/NuGet.Core/NuGet.Packaging/Signing/Utility/CertificateUtility.cs

```diff
@@ -28,7 +28,7 @@ public static class CertificateUtility
         public static string X509Certificate2ToString(X509Certificate2 cert, HashAlgorithmName fingerprintAlgorithm)
         {
             var certStringBuilder = new StringBuilder();
-            X509Certificate2ToString(cert, certStringBuilder, fingerprintAlgorithm, indentation: "");
+            X509Certificate2ToString(cert, certStringBuilder, fingerprintAlgorithm, indentation: "  ");
```

Comment:
I added indentation to all certs to match the output from signtool, see example in https://github.com/nuget/home/issues/6005#issuecomment-349099109